utils/manifests/utils.go

```diff
@@ -193,6 +193,19 @@ func deleteFile(path string) error {
 	return nil
 }
 
+// This helps in formating the leftover fields using a pre-defined dictionary
+func useDictionary(input string) string {
+	dict := map[interface{}]interface{}{ // includes Whitelist words
```

Comment:
why not just map[string]string ?

Reply:
I thought with that we can supply more general types. But using map[string]string would also be fine as we would just be using strings types.
I'll make that change.